frontend/src/views/SecretOverviewPage/SecretOverviewPage.store.tsx

```diff
@@ -0,0 +1,78 @@
+import { createContext, ReactNode, useContext, useEffect, useRef } from "react";
+import { useRouter } from "next/router";
+import { createStore, StateCreator, StoreApi, useStore } from "zustand";
+
+export enum EntryType {
```

Comment:
No need to for zustand. Simple react state itself will do the job. 

Zustand was used for dashboard for a different purpose

Reply:
resolved ^